package.json

```diff
@@ -164,7 +165,7 @@
     "yuidocjs": "0.10.2"
   },
   "engines": {
-    "node": "10.* || >= 12"
+    "node": ">= 10.10.0 || >= 12"
```

Comment:
Hmm, in general we can't don't change engines 🤔. Why is this needed?

Reply:
to use the http2 module for our tests we needed to bump this as it wasn't released to public usage until 10.10.0

Reply:
But we don't actually test locally with Node < 10.10, so it shouldn't really matter